guides/micronaut-metrics-oci/src/test/resources/application-test.yml

```
@@ -0,0 +1,7 @@
+crypto:
+  initialDelay: 10h # <1>
```

```suggestion
  initialDelay: 10h
```

Callout is not shown